### PR TITLE
Make OveruseOfNoqaCommentViolation configurable.

### LINKED ISSUE #820: Make `OveruseOfNoqaCommentViolation` configurable
Some people just use `noqa` for all the things. They write wrong code for no reason and later just put `noqa` to ignore it.

We need to be strict here: some projects totally require that `noqa` limit should be lower.
So, I guess we should make this configurable in the `[0, 10]` space.

### PR DESCRIPTION
**Pull request by lensvol**
As in issue, so on PR.

## Checklist

- [X] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [X] I have created at least one test case for the changes I have made
- [X] I have updated the documentation for the changes I have made
- [X] I have added my changes to the `CHANGELOG.md`

## Related issues

Closes #820

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ We used to have incremental versioning before `0.1.0`.
 
 - Forbids using `Literal[None]` in function annotations
 - Forbids use of vague import names (e.g. `from json import loads`)
+- Make **OveruseOfNoqaCommentViolation** configurable (`--max-noqa-comments`)
```
Comment (sobolevn) [inline CHANGELOG.md]: We use double backquotes, not bold text

File: tests/test_visitors/test_tokenize/test_comments/test_noqa_count.py [hunk 2]
```diff
@@ -31,3 +32,20 @@ def test_noqa_overuse(
...
+    parse_tokens,
+    assert_errors,
+    options,
+):
+    """Ensures that `noqa` overuse can be configured by options."""
+    file_tokens = parse_tokens(
+        'wallet = 10  # noqa: WPS002, WPS114\n' * defaults.MAX_NOQA_COMMENTS,
+    )
+
+    options = options(max_noqa_comments=defaults.MAX_NOQA_COMMENTS - 1)
```
Comment (sobolevn) [inline tests/test_visitors/test_tokenize/test_comments/test_noqa_count.py]: Let's also test `0` case: when no `noqa` is allowed

File: wemake_python_styleguide/options/config.py [hunk 3]
```diff
@@ -110,6 +110,12 @@
     defaults to
     :str:`wemake_python_styleguide.options.defaults.NESTED_CLASSES_WHITELIST`
 
+.. rubric:: Comments options:
+
+- ``max-noqa-comments`` - maximum number of `noqa` allowed in a module,
```
Comment (sobolevn) [inline wemake_python_styleguide/options/config.py]: It can fit into `General` section

**Comment by lensvol** [comment]
Implemented requested changes.

**Comment by lensvol** [comment]
Thank you for the opportunity! Now I can go back into hibernation for yet another year :)

**Comment by sobolevn** [comment]
@lensvol see you! 🙂

**Pull request merged**